spring-cloud-huawei-governance/src/main/resources/component.yaml

```diff
@@ -0,0 +1,28 @@
+#
+## ---------------------------------------------------------------------------
+##
+## Copyright (C) 2020-2024 Huawei Technologies Co., Ltd. All rights reserved.
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+
+x-component-order: 110
+
+feign:
+  client:
+    config:
+      default:
+        connectTimeout: 1000
+        readTimeout: 30000
+
+
```

Comment:
It's better to use spring-cloud-huawei-common component.yaml, no need add a new file.

Reply:
ok，fixed